src/components/index/animated-elements/hubblr-icon-image/AnimatedHubblrIconImage.jsx

```diff
@@ -3,9 +3,13 @@ import PropTypes from 'prop-types';
 import { motion } from 'framer-motion';
 
 function HubblrIconImage({ className }) {
+  const initial = { pathLength: 0, opacity: 0 };
```

Comment:
Konstanten machen wir generell noch in all caps snake case (also hier INITIAL oder sonst INITIAL_STH)